ELLog/Logger.swift

```diff
@@ -60,11 +60,11 @@ public struct LogLevel: OptionSetType, BooleanType, CustomStringConvertible {
             options.append("VERBOSE")
         }
 
-        return options.joinWithSeparator(", ")
+        return options.joined(separator: ", ")
     }
 
     public init(rawValue: UInt) { self.rawValue = rawValue }
-    
+    //todo WARNING BooleanType has been removed
```

Comment:
I think we can just remove the `boolValue` property. I can't think of a practical use case where comparing a log level as boolean would ever make sense.